src/propagate_constant.cpp

```diff
@@ -38,17 +38,22 @@ MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_PROPAGATE_CONSTANT)
 
 bool skip_propagate(instruction_ref ins)
 {
-    if(ins->name() == "contiguous")
+    if(ins->name() == "contiguous" or ins->name() == "dequantizelinear")
         return skip_propagate(ins->inputs().front());
+
     auto&& s = ins->get_shape();
+
+    if(ins->name() == "unpack_int4")
+        return true;
+
```

Comment:
Would this work if let's say there is a "reshape" or some other operation in between `unpack_int4` and `dequantizelinear` ? 
e.g. `unpack_int4 --> convert --> DQ`

Reply:
Why would there be a convert?

Reply:
Contrived case i came up with.

Need to handle general case if possible pre-emptively.

Reply:
It can any other op. "convert' is just an example

Reply:
We are the ones that will insert these operators either in quantize_int4 or in the onnx parser, so i dont understand why we would insert a `convert`, since it wouldn't work.

Reply:
model could also come from quantizer tool e.g. vitisAI quantizer.

Reply:
I will mark this resolved for now. Don't see any reshape after `unpack_int4`. Thanks.